proto/gen/go/dream11/od/auth/v1/auth_grpc.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go-grpc. DO NOT EDIT.
 // versions:
 // - protoc-gen-go-grpc v1.3.0
-// - protoc             v5.26.1
+// - protoc             v3.20.3
```

Comment:
install the latest protoc version with `brew install protobuf` and generate the proto files again

Reply:
done